### ISSUE TITLE
False Positives with Custom Classifier Training on Version 2.4.0

### ISSUE THREAD

**Issue by Kalachuchi07** [birdnet-team/BirdNET-Analyzer#886]
I am currently training a classifier on version 2.4.0 but I’m running into the same issue described in #791, where I’m getting a lot of false positives. The change log for 2.4.0 said that this has been fixed but the issue still seems to persist. I have tried training a classifier on the calls of the Isabela Oriole on 1.5.1, which worked fine. I even tested the custom classifier with recordings of a different bird (Philippine Nightjar) and there were minimal to no false positives (below is an example).

<img width="975" height="254" alt="Image" src="https://github.com/user-attachments/assets/de45aea2-e399-4f8e-a50b-a7a8dc491b39" />

However, when I try training a classifier on 2.4.0 with the exact same data and then test it with the same nightjar recording (that does not have any Isabela Oriole calls in it), I get this:

<img width="975" height="414" alt="Image" src="https://github.com/user-attachments/assets/2025dbd6-4b4a-4ee2-b269-472a56059c3b" />

Interestingly, if I run the 1.5.1 classifier on version 2.4.0 – again, on the same nightjar recording – a few false positives crop up that didn’t appear when I ran it on 1.5.1. I used all the default settings in both versions to create the custom classifier and to run the single analysis. 

Is there a possible solution for this, or can this be patched in the next version? I would really prefer to use the latest versions instead of relying on an older one. Thank you!

**Comment by max-mauermann** [comment]
Hello,
You mentioned that you used default settings in both versions.
The default learning rate has also been changed between these versions, so both classifier might not be trained with the same parameters.
Could you also train a classifier on 2.4.0 with the default learning rate value from 1.5.1 (which should be 0,001) and check if this improves the classifier?
In general we also recommend using the autotune option to automatically optimize the hyperparameters and not rely on the default parameters.
